distro/configuration/concepts/misc-core_data.csv

```diff
@@ -1,6 +1,4 @@
 Uuid,Void/Retire,Same as mappings,Fully specified name:en,Fully specified name:fr,Fully specified name:es,Short name:en,Short name:fr,Short name:es,Description:en,Description:fr,Description:es,Data class,Data type,Answers,_version:1,_order:1000
-1065AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA,,CIEL:1065;PIH:1065;AMPATH:1065;SNOMED CT:373066001,Yes,Oui,Sí,,,,Generic answer to a question.,,,Misc,N/A,,,
```

Comment:
Why are we removing these concepts?

Reply:
These are duplicates and already come with the OCL packages.